packages/circuit-ui/components/Popover/Popover.tsx

```diff
@@ -138,6 +128,20 @@ export const Popover = forwardRef<HTMLDialogElement, PopoverProps>(
     const animationDuration = isMobile ? 300 : 0;
     const prevOpen = usePrevious(isOpen);
 
+    const sizeOptions: SizeOptions = useMemo(
+      () => ({
+        padding: boundaryPadding,
+        apply({ availableHeight, elements }) {
+          elements.floating.style.setProperty(
+            '--popover-max-height',
+            `${availableHeight}px`,
+          );
+        },
+        boundary: document.documentElement,
```

Comment:
`useMemo` runs on the server, doesn't it? I'd add a condition here: 
```suggestion
        boundary: isServer ? undefined : document.documentElement,
```

Reply:
TIL !